Automattic/wp-calypso
# Add support for SitePlan.introductory_offer_raw_price_integer

## Proposed Changes

* This PR adds client-side support for a new field in the Site Plans API response to represent the integer price for introductory plan offers.
* The new field is being added on the back end in D154373-code

## Why are these changes being made?


* The goal of the new field is to make it possible to perform the same currency formatting for prices from introductory offers as we do for the standard plan prices. Without this field, we can't perform formatting using integer price values for introductory plan prices.

## Testing Instructions



* Ensure that you're calling an API where D154373-code has been applied
* You'll likely need to add debugging to your preferred consumer of the Site Plans API, and verify that the new field is being returned in the objects.
  - I may need some help with specifics here, maybe from @merkushin or @chriskmnds, as I may not have time to verify this before I am away for a week

## Pre-merge Checklist
